airflow/models.py

```diff
@@ -3882,7 +3924,7 @@ class Chart(Base):
         "User", cascade=False, cascade_backrefs=False, backref='charts')
     x_is_date = Column(Boolean, default=True)
     iteration_no = Column(Integer, default=0)
-    last_modified = Column(DateTime, default=func.now())
+    last_modified = Column(UtcDateTime, default=func.now())
```

Comment:
The migration script currently does not include this field `last_modified`, so it's still a `DATETIME` object, maybe we want to modify it to timestamp as well

Reply:
Good catch. I'll fix that

Reply:
Done